engine/dlib/src/remotery/lib/Remotery.c

```diff
@@ -5689,6 +5685,9 @@ static rmtError InitThreadSampling(ThreadProfilers* thread_profilers)
     }
 #endif
 
+    // Kick-off the background thread that watches for new threads
+    rmtTryNew(rmtThread, thread_profilers->threadGatherThread, GatherThreadsLoop, thread_profilers);
+
```

Comment:
Updating to this version fixes an issue we had with unit tests shutting down quickly, while the Remotery system was starting up.